app/assets/javascripts/components/shared/ModalDialog.tsx

```diff
@@ -30,17 +30,20 @@ export const ModalDialog: FunctionComponent = ({ children }) => {
 
 export const ModalDialogLabel: FunctionComponent<{
   closeDialog: () => void;
-}> = ({ children, closeDialog }) => (
-  <AlertDialogLabel className="">
-    <div className="px-4 py-4 flex flex-row items-center">
-      <div className="flex-grow color-text text-lg font-bold">{children}</div>
-      <IconButton
-        focusable={true}
-        title="Close"
-        className="color-neutral h-5 w-5"
-        icon="close"
+  className?: string;
+}> = ({ children, closeDialog, className }) => (
+  <AlertDialogLabel className={className}>
+    <div className="w-full flex flex-row justify-between items-center">
+      <div className="flex-grow color-text text-base font-medium">
+        {children}
+      </div>
+      <div
+        tabIndex={0}
+        className="font-bold color-info cursor-pointer"
         onClick={() => closeDialog()}
```

Comment:
```suggestion
        onClick={closeDialog}
```

Reply:
Please also remove `IconButton` import from "shared/ModalDialog.tsx" file, seems it's not used

Reply:
Good catch!